api/src/cache.rs

```diff
@@ -229,22 +197,24 @@ pub async fn warm_cluster_stats_cache(
 
     Ok(())
 }
-pub async fn warm_scores_cache(
-    context: &WrappedContext,
-    redis_client: &Arc<RwLock<redis::Client>>,
-    redis_tag: String,
-) -> anyhow::Result<()> {
-    info!("Loading scores from Redis");
+pub async fn warm_scores_cache(context: &WrappedContext) -> anyhow::Result<()> {
+    info!("Loading scores from DB");
     let warmup_timer = Instant::now();
 
-    let mut conn = redis_cache::get_redis_connection(redis_client).await?;
-    let mut tagged_key = format!("{}_scores", redis_tag);
-    let scores_json: String = conn.get(tagged_key).await?;
-    let scores: HashMap<String, ValidatorScoreRecord> = serde_json::from_str(&scores_json)?;
-    tagged_key = format!("{}_scores_all", redis_tag);
-    let multi_run_scores_json: String = conn.get(tagged_key).await?;
-    let multi_run_scores: HashMap<Decimal, Vec<ValidatorScoreRecord>> =
-        serde_json::from_str(&multi_run_scores_json)?;
+    let last_scoring_run =
+        store::utils::load_last_scoring_run(&context.read().await.psql_client).await?;
+    let scores = match &last_scoring_run {
+        Some(scoring_run) => {
+            store::utils::load_scores(
+                &context.read().await.psql_client,
+                scoring_run.scoring_run_id,
+            )
+            .await?
+        }
+        None => Default::default(),
+    };
+    let multi_run_scores =
+        store::scoring::load_all_scores(&context.read().await.psql_client).await?;
```

Comment:
The fact that we have most of scores loading in `store::utils` but this is in `store::scoring` caught my attention, but considering the overall state I don't think we need to refactor this, definitely not in this PR.

Reply:
I want to do some major re-factor of the structure and how queries are executed but agreed, not here.